snapcraft.yaml

```diff
@@ -29,6 +30,9 @@ description: |
        - read/write to gsettings:
          plugs: [gsettings, home]
          (note that the home plug is needed to read new value)
+       - use of the shared platform snap content:
+         plugs: [ubuntu-app-platform]
+         (content interface plug to use ubuntu-app-platform slot needed)
```

Comment:
I would add here the example you provided:
```
plugs:
  ubuntu-app-runtime:
      content: ubuntu-app-runtime
      interface: content
      target: ubuntu-app-runtime
```
Indeed, we can't in cloud parts (maybe open a bug for this?) share standard plugs definition. So we need to document it.

A note (and that will impact your content interface as well), please please, version your content interface. It's the only way to ensure that we can't get into a breakage after an API or ABI changes. So I would suggest `content: ubuntu-app-runtime1` (or whatever version). The content is what you version on both sides.